lib/game_start.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module GameStart
+  DIFFICULTY_LEVEL = {
+    easy: { attempts: 15, hints: 3 },
+    medium: { attempts: 10, hints: 2 },
+    hell: { attempts: 5, hints: 1 }
+  }.freeze
+
+  NOT_YET = '-'
+  GOT_IT = '+'
+
+  def make_number(chars = 4, numbers = 6)
+    (1..chars).map { rand(1..numbers) }.join
+  end
+
+  def check_numbers(secret, numbers)
+    minuses = (secret & numbers).map { |element| [secret.count(element), numbers.count(element)].min }.sum
```

Comment:
let's rework this using `zip`
```
secret.zip(numbers).each do |secret_element, numbers_element|
  # check array elements and add + or -
end
```

Reply:
i don't know how to do it so that it will give the right results.

Reply:
pseudocode:
```
secret.zip(numbers).each do |secret_element, numbers_element|
  next unless secret.include? numbers_element

  next add_plus if secret_code_number == user_code_number

  add_minus
end
```